src/elements/partners-block.html

```diff
@@ -5,8 +5,6 @@
 <link rel="import" href="../../bower_components/iron-icon/iron-icon.html">
 
 <link rel="import" href="../redux-mixin.html">
-<link rel="import" href="./shared-styles.html">
```

Comment:
I do. @robdodson told me that, but it was early 2015. We have to ask about this in Polymer slack channel